css/style.css

```diff
@@ -233,10 +233,18 @@
     height: 0;
   }
 
-  #app.connected #commands {
+  #app.connected #commands, #app.gererating #commands {
     height: 150px;
   }
 
+  #app.connected li.offline {
+    display: none;
+  }
+
+  #app.gererating li.online {
```

Comment:
Is `gererating` a typo for `generating`? (it's matched in the `.js` file as well if you decide to change it)

Reply:
Yep, that's a typo. Good catch.